storage/storage.go

```diff
@@ -29,40 +29,35 @@ import (
 type Store interface {
 	InsertUser(user *model.User) error
 	GetUser(id uuid.UUID, user *model.User) error
+	UpdateUser(user *model.User) error
 
 	CreateProject(
 		proj *model.Project,
-		ttpl []*model.TransactionTemplate,
-		stpl []*model.ScriptTemplate,
+		files []*model.File,
 	) error
 	UpdateProject(input model.UpdateProject, proj *model.Project) error
 	UpdateProjectOwner(id, userID uuid.UUID) error
 	UpdateProjectVersion(id uuid.UUID, version *semver.Version) error
 	ResetProjectState(proj *model.Project) error
 	GetProject(id uuid.UUID, proj *model.Project) error
+	GetAllProjectsForUser(userID uuid.UUID, proj *[]*model.Project) error
+	DeleteProject(id uuid.UUID) error
 
-	InsertAccount(acc *model.Account) error
-	InsertAccounts(accs []*model.Account) error
-	GetAccount(id, pID uuid.UUID, acc *model.Account) error
-	GetAccountsForProject(projectID uuid.UUID, accs *[]*model.Account) error
-	DeleteAccount(id, pID uuid.UUID) error
-	UpdateAccount(input model.UpdateAccount, acc *model.Account) error
+	InsertCadenceFile(file *model.File) error
+	UpdateCadenceFile(input model.UpdateFile, file *model.File) error
+	DeleteCadenceFile(id uuid.UUID, pID uuid.UUID) error
+	GetFile(id uuid.UUID, pID uuid.UUID, file *model.File) error
+	GetFilesForProject(projectID uuid.UUID, files *[]*model.File, fileType model.FileType) error
+	GetAllFilesForProject(projectID uuid.UUID, files *[]*model.File) error
```

Comment:
I think this should be consistently named, either `InsertCadenceFile` and `GetCadenceFile` or just `InsertFile` and `GetFile`.

Reply:
Done